drush.php

```diff
@@ -43,11 +43,16 @@ function drush_main() {
 
   // Find the selected site based on --root, --uri or cwd, and
   // return the bootstrap object that goes with it.
```

Comment:
These different preflight phases were split up, because dirty legacy stuff was happening in drush_main() between them.  Now, all that stuff has been cleaned  up, so I rolled all of these into drush_preflight, and made it return the bootstrap object again.